docs/marionette.region.md

```diff
@@ -257,18 +261,22 @@ This can be useful in unit testing your views.
 Override the region's `attachHtml` method to change how the view is attached
 to the DOM. This method receives one parameter - the view to show.
 
-The default implementation of `attachHtml` is:
+The default implementation of `attachHtml` works a bit like the following:
 
 ```javascript
 var Mn = require('backbone.marionette');
 
 Mn.Region.prototype.attachHtml = function(view){
-  this.$el.empty().append(view.el);
+  if (this.isShowingAView()) {
```

Comment:
I think the best example would simply be:
```javascript
Mn.Region.prototype.attachHtml = function(view){
  this.el.appendChild(view.el);
});
```

But we should probably document that if you change the default `attachHtml` then you cannot use `replaceElement`